pygmt/clib/session.py

```diff
@@ -717,7 +717,9 @@ def _check_dtype_and_dim(self, array, ndim):
         """
         # check the array has the given dimension
```

Comment:
Upper-case beginning for consistency, together with the suggestion by @michaelgrund.
```suggestion
        # Check that the array has the given number of dimensions.
```